### PR TITLE
fixes #17534 - server_ssl_crl should be blank string not boolean

### PR DESCRIPTION
**Pull request by stbenjam**
Needs https://github.com/theforeman/puppet-foreman/pull/509

### CODE REVIEW

**Comment by stbenjam** [comment]
Oh look, another ridiculous cop from rubocop

**Comment by stbenjam** [comment]
puppet-foreman PR merged

File: Gemfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  gem 'rubocop'
   gem 'rspec'
+  gem 'rubocop'
 end
 
 gem 'kafo', '>= 1.0.1'
-gem 'rake'
-gem 'puppet', '>= 3.6.0', '< 4.0.0'
 gem 'librarian-puppet', '~> 2.2.0'
+gem 'puppet', '>= 3.6.0', '< 4.0.0'
```
Comment (ekohl) [inline Gemfile]: Could this be increased to ```< 5.0.0``` now?

File: config/katello.migrations/161130090556-server_ssl_crl_blank.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# server_ssl_crl is expected to be an absolute path or a blank string, not
+# false. Katelllo isn't using a CRL, so this needs to be a blank string.
+answers['foreman']['server_ssl_crl'] = ""
```
Comment (stbenjam) [inline config/katello.migrations/161130090556-server_ssl_crl_blank.rb]: Yea, you're right, I'll check first if it has a value other than false: https://github.com/Katello/katello-installer/pull/444

File: Gemfile (hunk 1 above)
Comment (ekohl) [inline Gemfile]: Yes, I was just wondering.

**Pull request merged**